xspec/xspec
# Fix issue 208 (Non alphanumeric path can make code coverage fail)

This pull request derives from #343. So needs to be handled after that.

---

Fixes #208.

The root cause of the problem is that XSLTCoverageTraceListener outputs XML as string.
This pull request uses Saxon's XML serializer instead.

#### How to test this pull request

1. Check out fffbb604e51c43a09be499ed1d9f19bdf236ce5e which reproduces the problem.

1. Go to `[XSpec]/test`.
1. Do `export SAXON_JAR=/path/to/saxon9ee.jar`. (Saxon-PE should be fine too, but I haven't tested.)
1. Run `./run-bats.sh --tap`.
    ==> Test 14 should fail.

1. Check out 4345c2ed5ddf077cdae744e379438f576a96803c which fixes the problem.

1. Run `./run-bats.sh --tap`.
    ==> Test 14 should pass.

Do the same steps with `run-bats.cmd` (with no arguments).
